### DISCUSSION TITLE
Welcome to RaspAP Discussions!

### DISCUSSION THREAD

**Discussion by billz** [RaspAP/raspap-webgui#729] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the RaspAP community. We hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we build together 🛠

  To get started, comment below with an introduction of yourself and tell us about how you use RaspAP.

**Comment by EvilOlaf** [comment]
Interesting feature.

**Comment by vhDavid** [comment]
Hi, I don't use RaspAP... yet... I am a volunteer in a visitor-center and am looking for wifi-hotspot software that has the ability to change the wifi-password automatically. The idea is to let our website generate a password at night with a chronjob and store it in a database. Next let the hotspot read it at boot (maybe a url in the config file instead of a password) and set it as wifi password and another pi also read it at boot and display it on a screen. Another option is that the hotspot software generates a password at boot and let another pi read it somehow at boot and display it. Is that possible with RaspAP in any way? Maybe a mod I can do myself?

**Reply by billz** [reply]
@vhDavid please start a new discussion in Q&A, thanks
